### PR TITLE
e2e: parallelize end-to-end tests

### PR DESCRIPTION
**Pull request by hiddeco**

### CODE REVIEW

**Comment by 2opremio** [comment]
You are missing setting the number of clusters when invoking `make e2e` from CircleCI

**Comment by hiddeco** [comment]
@2opremio I think you were looking at an old commit.

File: .circleci/config.yml [hunk 1]
```diff
@@ -52,7 +53,16 @@ jobs:
...
+      - run:
+          name: End-to-end Helm v2
+          command: E2E_KIND_CLUSTER_NUM=4 make e2e
+          environment:
+            HELM_VERSION: v2
+      - run:
+          name: End-to-end Helm v3
+          command: E2E_KIND_CLUSTER_NUM=4 make e2e
+          environment:
+            HELM_VERSION: v3
```
Comment (2opremio) [inline .circleci/config.yml:65]: Does it make sense to both pass the env variables through the shell and through `environment`? I would go for one of the two mechanisms

**Comment by 2opremio** [comment]
> @2opremio I think you were looking at an old commit.

I think you may be right, it looks good now

File: .circleci/config.yml (hunk 1 above)
Comment (hiddeco) [reply .circleci/config.yml:65]: I used the `environment` for the Helm version to highlight the difference between the two `run` items.

Comment (2opremio) [reply .circleci/config.yml:65]: alright

**Review by 2opremio — APPROVED** [review]

**Pull request merged**